lualib/lua_scanners/icap.lua

```diff
@@ -240,10 +240,19 @@ local function icap_check(task, content, digest, rule, maybe_part)
 
       local function get_req_headers()
 
+        local in_client_ip = task:get_from_ip()
+        lua_util.debugm(rule.name, task, 'URL: http://%s/%s | Content-Type: %s/%s',
```

Comment:
This will crash the plugin when `maybe_part` is nil. Lua does not do lazy evaluations. Also, that will call two C functions twice (get_filename and get_detected_type).

Reply:
Didn't notice I am missing a check before the debug message. Is it OK to just remove it?

Reply:
Just removed the debug message